test/asami/query_internals_test.cljc

```diff
@@ -363,4 +363,14 @@
       (is (vartest? p))
       (is (vartest? v)))))
 
+(deftest test-column-renaming
```

Comment:
This serves as an integration test of sorts between the query and project layers. I think this is a bit indirection and would be more comfortable if

```clj
(partial project internal/project-args)
```

were defined in the query namespace, say, as `default-project`, and then substituted in `asami.query/query-entry` and throughout this namespace.